src/preamble.js

```diff
@@ -2223,6 +2223,7 @@ function integrateWasmJS(Module) {
       exports = instance.exports;
       if (exports.memory) mergeMemory(exports.memory);
       Module['asm'] = exports;
+      asm = exports;
```

Comment:
Sounds good, reverted in https://github.com/kripken/emscripten/pull/5099/commits/63f2f813b313d6e6303a3c40891c4155312a499b